Update bare template (#8)
scale-it/algo-builder-templates: bare/README.md

# Algo Builder Web DApp

> This is a boilerplate react and algob project which serves as a template for Dapp using algo builder. It has a single package.json file. `algob` works within the project (using `algob deploy`).

Reviewer: ```suggestion
This is a React + `algob` project which serves as a app template. It has a single package.json file. `algob` works within the project (using `yarn run algob deploy`).
```

Reviewer: Maybe we need to write that a user will need to add Algo Builder using:
* `yarn add @algo-builder/algob`
* or `yarn link @algo-builder/algob`

WDYT?

Author: Yes, we should do that (using `yarn link` is recommended). It's on my mind when we'll update the main documentation of templates. However, i will add it here as well.
